crates/relay/src/benchmarking.rs

```diff
@@ -157,10 +166,26 @@ benchmarks! {
         BtcRelay::<T>::store_block_header(&relayer_id, block_header).unwrap();
         Security::<T>::set_active_block_number(Security::<T>::active_block_number() + BtcRelay::<T>::parachain_confirmations() + 1u32.into());
 
-        ExchangeRateOracle::<T>::_set_exchange_rate(
+        ExchangeRateOracle::<T>::_set_exchange_rate(DEFAULT_TESTING_CURRENCY,
             <T as exchange_rate_oracle::Config>::UnsignedFixedPoint::one()
         ).unwrap();
     }: _(RawOrigin::Signed(origin), vault_id, proof, raw_tx)
 }
 
 impl_benchmark_test_suite!(Relay, crate::mock::ExtBuilder::build_with(|_| {}), crate::mock::Test);
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::mock::{ExtBuilder, Test};
+    use frame_support::assert_ok;
+
+    #[test]
+    fn test_benchmarks() {
+        ExtBuilder::build().execute_with(|| {
+            assert_ok!(test_benchmark_initialize::<Test>());
+            assert_ok!(test_benchmark_store_block_header::<Test>());
+            assert_ok!(test_benchmark_report_vault_theft::<Test>());
+        });
+    }
+}
```

Comment:
I think this is a duplication of `impl_benchmark_test_suite!`

Reply:
Woops, that was a leftover from debugging